primer/src/Primer/Eval/Redex.hs

```diff
@@ -181,7 +181,8 @@ import Primer.Zipper.Type (
 
 {- HLINT ignore ViewRedexOptions "Use newtype instead of data" -}
 data ViewRedexOptions = ViewRedexOptions
-  { pushMulti :: Bool
+  { pushMulti :: Bool -- TODO/REVIEW: rename this: groupedLets?
```

Comment:
One last bikeshedding thought: I'm not particularly happy calling this `pushMulti`, since it will also affect elision -- if we are aggressively eliding, then with `pushMulti == True` we will elide everything from a group rather than one-at-a-time.

Any better ideas for names?

Reply:
I think `groupedLets` is better.